infrastructure/README.md

```diff
@@ -17,6 +17,7 @@ For the GCP project where the infra resources are being created, the following p
 Following service APIs are enabled, 
 - container.googleapis.com
 - gkehub.googleapis.com
+- servicenetworking.googleapis.com
```

Comment:
Also add cloudresourcemanager.googleapis.com please

Reply:
@umeshkumhar can we consider enabling these automatically in the `infrastructure` module, similar to what we do with `rag`, `ray,` and `jupyter`?

Reply:
https://github.com/GoogleCloudPlatform/ai-on-gke/issues/474